components/UnstructuredTab/UnstructuredTab.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { Svg } from 'react-optimized-image';
+import { StyledTabButton } from 'components/Tab/Tab';
+import { Col } from 'sections/gov/components/common';
+import InfoIcon from 'assets/svg/app/info-pink.svg';
+
+export type TabInfo = {
+	title: string;
+	description: string;
+	tabChildren: ReactNode;
+	key: string;
+	blue: boolean;
+	disabled?: boolean;
+};
+
+export type UnstructuredTabProps = {
+	tabData: TabInfo;
+};
+
+const StyledTabButtonExtended = styled(StyledTabButton)`
+	flex-direction: row;
+	justify-content: space-between;
+	padding: 0px 20px;
+`;
```

Comment:
Since we cant click on this button when it's an "UnstructedTab", it might be worth adding `cursor: default`